src/Microsoft.AspNet.Mvc.Razor/Resources.resx

```diff
@@ -148,7 +148,7 @@
     <value>There is no active writing scope to end.</value>
   </data>
   <data name="RazorPage_YouCannotFlushWhileInAWritingScope" xml:space="preserve">
-    <value>You cannot flush while inside a writing scope.</value>
+    <value>You cannot flush while inside a writing scope. (Path: '{0}')</value>
```

Comment:
(And change the string name while we're at it - the word `you` is so weird in an error message.)